bootstraptest/test_insns.rb

```diff
@@ -214,8 +214,21 @@ def freeze
     'true'.freeze
   },
 
+  [ 'opt_duparray_send', %q{ x = :a; [:a, :b].include?(x) }, ],
+  [ 'opt_duparray_send', <<-'},', ], # {
+    class Array
+      def include?(i)
+        i == 1
+      end
+    end
+    x = 3
+    [:a, :b].include?(1)
```

Comment:
Should this be `include?(x)`? If not, what's the assignment in the line above needed for?